testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -45,6 +45,7 @@
     SUPPORTS_JOIN_PUSHDOWN_WITH_VARCHAR_EQUALITY(fallback -> fallback.test(SUPPORTS_JOIN_PUSHDOWN) && fallback.test(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY)),
     SUPPORTS_JOIN_PUSHDOWN_WITH_VARCHAR_INEQUALITY(fallback -> fallback.test(SUPPORTS_JOIN_PUSHDOWN) && fallback.test(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY)),
     SUPPORTS_CREATE_SCHEMA,
+    SUPPORTS_RENAME_SCHEMA(false),
```

Comment:
Do more connectors support this or do most do not? i.e. let's pick a default which requires the least number of explicit `case SUPPORTS_RENAME_SCHEMA: return true/false`

Reply:
The `renameSchema` is overridden only in Hive and Iceberg connectors, they seem to be good candidates - enabled Base and BaseSmoke tests only for them.